python/taichi/lang/ops.py

```diff
@@ -439,17 +461,21 @@ def append(l, indices, val):
 
 
 def is_active(l, indices):
+    _taichi_skip_traceback = 1
```

Comment:
Personally i don't think this should be skipped, because these ops are public APIs (IIUC, originally this is to skip some internal implementation details like decorators?) But I don't feel strong about my opinion, your call..